websites/K/KeyFM/dist/metadata.json

```diff
@@ -0,0 +1,55 @@
+{
+  "author": {
+    "name": "Jack Hansen",
+    "id": "140590672657776641"
+  },
+  "service": "KeyFM",
+  "description": {
+    "en": "KeyFM.net is a non-profit radio community providing music around the globe, all day - every day."
+  },
+  "url": [
+    "keyfm.net"
+  ],
```

Comment:
its only 1 url now so it doesnt need to be in an array

Reply:
Good spot! Apologies for that one, I had to edit before heading to work. Shall edit that one now.